cli/src/helper.rs

```diff
@@ -343,3 +343,31 @@ pub async fn check_file_exists(path: &Path) -> Result<(), Error> {
     }
     Ok(())
 }
+
+pub enum SecretManagerChoice {
+    Stronghold,
+    LedgerNano,
+    LedgerNanoSimulator,
+}
+
+impl From<usize> for SecretManagerChoice {
+    fn from(value: usize) -> Self {
+        match value {
+            0 => Self::Stronghold,
+            1 => Self::LedgerNano,
+            2 => Self::LedgerNanoSimulator,
+            _ => unreachable!(),
```

Comment:
probably should use a `panic!` instead with a message, even if this shouldn't happen

Reply:
okay

Reply:
IMO `unreachable` should pretty much only be used when a branch is invalidated because of prior code *in the same scope*. This is a case where, in the context of this impl, the branch is definitely not invalidated.

Reply:
changed